doc/configuration/authentication_modes.rst

```diff
@@ -0,0 +1,133 @@
+.. _authentication_modes:
+
+Authentication Modes
+====================
+
+privacyIDEA supports a variety of tokens that implement different
+authentication flows. We call these flows *authentication modes*. Currently,
+tokens may implement three authentication modes, namely ``authenticate``,
+``challenge`` and ```outofband``.
+
+Application plugins need to implement the three authentication modes
+separately, as the modes differ in their user experience. For example:
+
+* The HOTP token type implements the ``authenticate`` mode, which is a
+  single-shot authentication flow. For each authentication request, the user
+  uses their token to generate a new HOTP value and enters it along with their
+  OTP PIN. The plugin sends both values to privacyIDEA, which decides whether
+  the authentication is valid or not.
+* The E-Mail token type implements the ``challenge`` mode. With an E-Mail token,
```

Comment:
"The E-Mail token or SMS-Token type impement"

Reply:
thanks! fixed in 076624be5af5d799ff9e8cdf6e6d362f1781e495